pkgs/flox-activations/default.nix

```diff
@@ -0,0 +1,94 @@
+{
+  inputs,
+  gnused,
+  pkgsFor,
+  rust-toolchain,
+  rustfmt ? rust-toolchain.rustfmt,
+  flox-src,
+  rust-external-deps,
+  rust-internal-deps,
```

Comment:
These are only used at build time and will never leak into the closure, right?

Reply:
The external deps are used at build time because we have workspace dependencies set up and this crate may share some of them. When I look at the deps for `flox-activations` it doesn't show any other packages, but it's possible I'm not measuring that correctly:
```
❯ nix-store --query --requisites /nix/store/3zw586h8l91rirlgrfja4j1zgfsp9bgp-flox-activations-1.3.2
/nix/store/3zw586h8l91rirlgrfja4j1zgfsp9bgp-flox-activations-1.3.2
```